docs/user-guide/host-settings.md

```diff
@@ -0,0 +1,194 @@
+# Host Settings
+This page contains settings for Linux servers, it agnostic of the Linux
+distribution. The commands use the `ip` utility from the `iproute2` package.
+These settings do not persist across reboots. Consult the documentation from
+the Linux distribution for guides on how to persist the settings in for example
+[Netplan](https://documentation.ubuntu.com/server/explanation/networking/configuring-networks/index.html) or 
+[Network Manager](https://networkmanager.dev/docs/admins/). For additional
+details consult the [kernel bonding
+driver](https://www.kernel.org/doc/html/latest/networking/bonding.html)
+documentation.
+
+## MCLAG / ESLAG
+
+The multi-chassis LAG architecture is a way to provide device redundancy
+in a network architecture. At the physical layer an MCLAG topology is a single
+server connected to two different switches and those switches are directly connected
+to each other in addition to being connected to the rest of the fabric.
+
+ESLAG is a similar topology as MCLAG with the beneficial difference that the
+switches do not need a to be connected to each other. There can be up to 4
```

Comment:
The switches do have to be connected to each other, but it can be through the spines, not a direct connection between the switches, we should clarify.  Perhaps a note that we implement the MCLAG of the underlying switch platform and provide a link to MCLAG generally?

Reply:
For ESLAG, we should have a link explaining how ESI Multihomeing works, but just a reference that ESLAG will use ESI MH for configuration and have a link.

Reply:
@Frostman can correct me if I am wrong, but I don't believe we support having an MCLAG communicate through the fabric. 

As far as links on MCLAG / ESLAG, outside of wikipedia, there isn't a lot of what I think are high quality links. Most of them are to vendor specific documentation, or a bloggers page. I can link to the sonic page, but that might create confusion.